az_bootstrap_release action is currently broken

## Problem/Motivation
The `az_bootstrap_release` GitHub action for Quickstart is currently broken:


### Describe the bug
https://github.com/az-digital/az_quickstart/runs/1875408952?check_suite_focus=true
```
Switched to a new branch 'az-bootstrap-2.0.6'
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <runner@fv-az59-891.dgk0fakyq5wefbmcp0h0z3jz1d.cx.internal.cloudapp.net>) not allowed
Error: Process completed with exit code 128.
```

--- a/.github/workflows/arizona-bootstrap-release.yml
+++ b/.github/workflows/arizona-bootstrap-release.yml
@@ -16,6 +16,8 @@ jobs:
 
       - name: Create new branch and PR
         run: |
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "${GITHUB_ACTOR}"
           git checkout -b 'az-bootstrap-${{ github.event.client_payload.version }}'
           jq '.require."az-digital/arizona-bootstrap" |= "${{ github.event.client_payload.version }}"' composer.json > composer.json.new
           mv composer.json.new composer.json

